scenarios/segmentation/01_training_introduction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>this is pretty cool</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>